src/picongpu/include/fields/background/cellwiseOperation.hpp

```diff
@@ -38,7 +38,7 @@ namespace cellwiseOperation
 
     /** Kernel that calls T_OpFunctor and T_ValFunctor on each cell of a field
      *
-     *  Pseudo code: opFunctor( cell, valFunctor( globalCellIdx, currentStep ) );
+     *  Pseudo code: opFunctor( cell, valFunctor( totalCellIdx, currentStep ) );
```

Comment:
Please revert or give a better name. This offset must be the offset to the origin of the global domain.